src/Microsoft.OData.Client/Materialization/EntryValueMaterializationPolicy.cs

```diff
@@ -534,11 +534,11 @@ private object GetOrCreateCollectionProperty(object instance, ClientPropertyAnno
             ClientEdmModel edmModel = this.MaterializerContext.Model;
             ClientTypeAnnotation collectionType = edmModel.GetClientTypeAnnotation(edmModel.GetOrCreateEdmType(property.ResourceSetItemType));
 
-            IEnumerable<ODataResource> entries = MaterializerFeed.GetFeed(feed).Entries;
+            IEnumerable<ODataResource> entries = MaterializerFeed.GetFeed(feed, this.MaterializerContext).Entries;

```

Comment:
Is `MaterializerFeed.GetFeed(feed, this.MaterializerContext)` equivalent to `this.MaterializerContext.GetAnnotatable<MaterializerFeed>(feed)`?

Reply:
Yes, it is. The method is kind of superfluous come to think of it. But I think it can be refactored later to avoid unnecessarily passing the context around too much.